Problems with the beat generator and mage

I ran into this issue while following the guide to make a beat. I looked around the internet and found the solution on the discuss forum. The solution posted there fixed my issue as well, so it seem like it's still a problem. I am opening this issue to see about getting it fixed or the documentation updated.

Please include configurations and logs if available.

```
$ python $GOPATH/src/github.com/elastic/beats/script/generate.py
Beat Name [Examplebeat]:
Your Github Name [your-github-name]: richbai90
Beat Path [github.com/richbai90/examplebeat]:
Firstname Lastname: Rich Baird
$ cd examplebeat/
$ python --version
Python 2.7.16
$ virtualenv --version
16.4.3
$ make setup
...
make[1]: *** No rule to make target `../dev-tools/make/mage.mk'.  Stop.
make: *** [pre-setup] Error 2
```

For confirmed bugs, please report:
- Version: latest? -- I'll update this when I know how to get that info
- Operating System: MacOS 10.13.6
- Discuss Forum URL: https://discuss.elastic.co/t/please-help-create-new-beats/163560
- Steps to Reproduce: Follow the guide on the official documentation https://www.elastic.co/guide/en/beats/devguide/current/newbeat-generate.html and run make setup

--- a/generator/beat/{beat}/Makefile
+++ b/generator/beat/{beat}/Makefile
@@ -18,7 +18,7 @@ NO_COLLECT=true
 setup: pre-setup git-add
 
 pre-setup: copy-vendor git-init
-	$(MAKE) -f $(LIBBEAT_MAKEFILE) mage
+	$(MAKE) -f $(LIBBEAT_MAKEFILE) mage ES_BEATS=$(ES_BEATS)
 	$(MAKE) -f $(LIBBEAT_MAKEFILE) update BEAT_NAME=$(BEAT_NAME) ES_BEATS=$(ES_BEATS) NO_COLLECT=$(NO_COLLECT)
 
 # Copy beats into vendor directory

